src/HealthChecks.AzureServiceBus/AzureEventHubHealthCheck.cs

```diff
@@ -17,38 +18,48 @@ public class AzureEventHubHealthCheck : IHealthCheck
 
     private string? _connectionKey;
 
-    private string ConnectionKey =>
-        _connectionKey ??= _connectionString ?? $"{_endpoint}_{_eventHubName}";
+    private string ConnectionKey => _connectionKey ??= _connectionString ?? $"{_endpoint}_{_eventHubName}";
 
-    public AzureEventHubHealthCheck(string connectionString, string eventHubName)
+    public AzureEventHubHealthCheck(AzureEventHubOptions options)
```

Comment:
Why doesn't this HC store options in a single field like others do?

Reply:
It was less effort this way 🙂 Although I can replace it with an options field, if you'd like(?).

Reply:
Yes, to encourage such design across codebase.